test/docgen/view/__snapshots__/documentation.test.ts.snap

```diff
@@ -55687,7 +55687,7 @@ aws_cdk.aws_ecr.CfnRepository(
                     "fqn": "constructs.Construct",
                     "id": "constructs.Construct",
                     "packageName": "constructs",
-                    "packageVersion": "3.3.187",
+                    "packageVersion": "3.3.188",
```

Comment:
Hrmmmmm. It looks like what's happening is that since most packages we generate have caret dependencies, the version of the referenced type isn't necessarily fixed.

I've been thinking about this for a while, including whether there's some better way to model this so that the package version stays fixed given the current assembly (e.g. by including the semver string from package.json instead) -- but it turns out that forces a lot of additional complexity onto downstream users.

For now I'm going to simply add some extra code that will make the snapshots stable by always linking to the same package version within unit tests (e.g. "1.2.3")